src/main/java/com/autotune/database/dao/ExperimentDAOImpl.java

```diff
@@ -27,6 +29,7 @@ public class ExperimentDAOImpl implements ExperimentDAO {
     public ValidationOutputData addExperimentToDB(KruizeExperimentEntry kruizeExperimentEntry) {
         ValidationOutputData validationOutputData = new ValidationOutputData(false, null, null);
         Transaction tx = null;
+        Timer.Sample timeraddExpDB = Timer.start(MetricsConfig.meterRegistry());
```

Comment:
we finding timer for entire createExperiment POST API which includes DB call
do we need extra timer inside dao classes